lib/tarpon/entity/subscriber.rb

```diff
@@ -7,7 +7,7 @@ class Subscriber
 
       def initialize(attributes = {})
         @raw          = attributes
-        @entitlements = EntitlementList.new(attributes[:entitlements])
+        @entitlements = EntitlementList.new(attributes[:entitlements] || {})
```

Comment:
Is the `entitlements` field from the subscriber object really nullable? Just tested fetching a subscriber without entitlements and it already returns an empty object.

Reply:
Yes, the field was empty in some cases. The subscriber object is not consistent unfortunately. I cannot recall exactly the scenario, but i was also kind of „shocked“ because the documentation is not mentioning this.

Reply:
Empty is fine, was it null though?

Reply:
The key was basically not present.

Reply:
Otherwise I would not have implemented the fallback with an empty hash

Reply:
Ah ok, empty key.
Right, I think it doesn't hurt having a fallback there.